redis-snapshot-replicator/main.tf

```diff
@@ -219,145 +224,149 @@ data "archive_file" "create_zip" {
 
 locals {
   # Solution from this comment to open issue on non-relative paths  # https://github.com/hashicorp/terraform/issues/8204#issuecomment-332239294
-  count    = "${var.enable ? 1 : 0}"
-  filename = "${substr(data.archive_file.create_zip.output_path, length(path.cwd) + 1, -1)}"
-
+  count = var.enable ? 1 : 0
+  filename = substr(
```

Comment:
I think this is not really needed anymore, the issue described above is resolved in TF 0.12 afaik.